src/iuse.cpp

```diff
@@ -2437,9 +2436,8 @@ cata::optional<int> iuse::hammer( player *p, item *it, bool, const tripoint & )
         type == t_rdoor_boarded || type == t_rdoor_boarded_damaged ||
         type == t_door_boarded_peep || type == t_door_boarded_damaged_peep ) {
         // pry action
-        player_activity act( ACT_PRY_NAILS, to_moves<int>( 30_seconds ), -1 );
-        act.placement = pnt;
-        p->assign_activity( act );
+        p->assign_activity( player_activity( pry_nails_activity_actor( to_moves<int>( 30_minutes ),
```

Comment:
I suppose that's just a typo? One doesn't need half an hour to pry out a handful of nails.
```suggestion
        p->assign_activity( player_activity( pry_nails_activity_actor( to_moves<int>( 30_seconds ),
```

Reply:
Yeah, it's a typo.